Refactor and various fixes

topless opened the pull request.

topless committed:
Fix for pydoc style

topless commented:
This is expected to fail since it cannot pul the `npm` package.

ntarocco commented:
> This is expected to fail since it cannot pul the `npm` package.

Ok then let's wait for the npm to be ready!

topless committed:
Refactor

- Replaced all cominations of invenio-record-editor to invenio-records-editor
- The project now points to inveniosoftware repository
- Replaced all author entries with inveniosoftware

topless committed:
bundles: point to newly released package

topless committed:
tests: override assets 

When testing we need to remove the `{{ ASSET_URL }}` because tests fail to find relevant css and js bundles.

topless commented:
- Replaced all cominations of invenio-record-editor to invenio-records-editor
- The project now points to inveniosoftware repository
- Replaced all author entries with inveniosoftware
- Fixes for testing when cannot find bundles

ntarocco commented on invenio_records_editor/views.py:
```diff
@@ -17,9 +18,9 @@
 def create_editor_blueprint(app):
```
Question, why this function?
We can just register the Blueprint in the entrypoints, without having to add the registration manually when creating the app.

ntarocco commented on RELEASE-NOTES.rst:
```diff
@@ -19,18 +19,18 @@ What's new
+   https://invenio-records-editor.readthedocs.io/
 
-Happy hacking and thanks for flying Invenio-Record-Editor.
+Happy hacking and thanks for flying Invenio-Records-Editor.
```
LoL, even if I personally like it, not sure we want to keep it ;)

ntarocco reviewed (commented):
Would it be possible to have the mock data in the example app and not in the main `src`?

topless committed:
global: Refactored extension

- Renamed InvenioRecordEditor to InvenioRecordsEditor.
- Included font-awesome from CDN.
- Overridable editor template.

topless committed:
release: v1.0.0a1
